bom/pom.xml

```diff
@@ -0,0 +1,138 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <artifactId>cucumber-jvm</artifactId>
+        <groupId>io.cucumber</groupId>
+        <version>6.10.1-SNAPSHOT</version>
+    </parent>
+
+    <artifactId>cucumber-bom</artifactId>
+    <packaging>pom</packaging>
+    <name>Cucumber-JVM: Bill of Materials</name>
+
+    <dependencyManagement>
+        <dependencies>
+            <dependency>
+                <groupId>io.cucumber</groupId>
+                <artifactId>cucumber-core</artifactId>
+                <version>${project.version}</version>
```

Comment:
What happens when use this project as a parent pom with a different project version?

Reply:
Getting an error:
```
[ERROR] Failed to execute goal on project coffee-boots-example: Could not resolve dependencies for project io.github.stepio.examples:coffee-boots-example:jar:0.0.1-SNAPSHOT: Could not find artifact io.cucumber:cucumber-core:jar:0.0.1-SNAPSHOT -> [Help 1]
```
Thanks, that's the case I was looking for, will play more with plan B.